lesson_28/db/db_app/src/main/resources/queries/amiyahjones.sql

```diff
@@ -0,0 +1,22 @@
+--  returns the counts of media items by type.
+SELECT 
+  type, COUNT(*) AS item_count
+FROM 
+  media_items
+GROUP BY 
+  type;
+
+-- returns the sum of total pages checked out by guests.
+SELECT 
+SUM(pages) AS total_pages
+FROM 
+guests;
```

Comment:
Should execute your select from the `media_items` table and filter checked out items using a `where` clause.